examples/stable/mysql/README.md

```diff
@@ -10,35 +10,47 @@ This chart bootstraps a single node MySQL deployment on a [Kubernetes](http://ku
 
 - Kubernetes 1.6+ with Beta APIs enabled
 - PV provisioner support in the underlying infrastructure
-- Kanister controller version 0.26.0 installed in your cluster
-- Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+- Kanister controller version 0.26.0 installed in your cluster, let's assume in Namespace `<kanister-ns>`
```

Comment:
```suggestion
- Kanister controller version 0.26.0 installed in your cluster, let's assume in Namespace `kanister`
```

Reply:
This will allow us to use the namespace in the examples below. Which makes it easy to copy paste